KitX Website Flutter/kitx_website/lib/pages/plugins_page.dart

```diff
@@ -18,14 +15,6 @@ class _PluginsPageState extends State<PluginsPage> {
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      appBar: AppBar(
-        title: Text('PluginsPage'),
-      ),
-      drawer: AppDrawer(),
-      drawerEnableOpenDragGesture: true,
-      drawerEdgeDragWidth: (window.innerWidth ?? 300) / 2 + 50,
-      body: Center(child: Text('Under developing ...')),
-    );
+    return Center(child: Text('Under developing ...'));
```

Comment:
Add a `// TODO: ` note here?

Reply:
It is not necessary for the time being, because this can be seen directly on the user interface